Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Install dependencies only when needed
+FROM node:16-alpine AS builder
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+COPY . .
+RUN yarn install --frozen-lockfile
```

Comment:
Will these files ever require changes? I don't really use Docker and I think more junior contributors also probably don't.

Reply:
This is related to deployment, so they won't need any changes, at least in the near future.